autoscheduler/frontend/src/components/CourseSelectColumn/CourseSelectColumn.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { useSelector, useDispatch } from 'react-redux';
+import { Button } from '@material-ui/core';
+import * as styles from './CourseSelectColumn.css';
+import { RootState } from '../../redux/reducers';
+import { CourseCardArray, CustomizationLevel } from '../../types/CourseCardOptions';
+import CourseSelectCard from './CourseSelectCard/CourseSelectCard';
+import { addCourseCard, removeCourseCard } from '../../redux/actions';
+
+/**
+ * Renders a column of CourseSelectCards, as well as a button to add course cards
+ */
+const CourseSelectColumn: React.FC<RouteComponentProps> = () => {
+  const courseCards = useSelector<RootState, CourseCardArray>(
+    (state) => state.courseCards,
+  );
+
+  const dispatch = useDispatch();
+
+  const rows: JSX.Element[] = [];
+
+  // Add all of the course cards to rows to be displayed
+  for (let i = 0; i < courseCards.numCardsCreated; i++) {
+    if (courseCards[i]) {
+      rows.push(
+        <div
+          className={styles.row}
+          key={`${i} stuff`}
```

Comment:
let's make this key `\`${course-card-i}\`` just to keep it looking a little more professional/descriptive

Reply:
Tbh I made this change but I think it got lost somewhere along the way when I was squashing/rebasing. Will add